zebra-network/src/peer/handshake.rs

```diff
@@ -409,7 +409,7 @@ where
             let (tx, _) = broadcast::channel(100);
             tx
         });
-        let timestamp_collector = self.timestamp_collector.unwrap_or_else(|| {
+        let address_book_updater = self.address_book_updater.unwrap_or_else(|| {
             // No timestamp collector was passed, so create a stub channel.
             // Dropping the receiver means sends will fail, but we don't care.
```

Comment:
Maybe update this comment as well?

```suggestion
            // No `AddressBookUpdater` for timestamp collection was passed, so create a stub
            // channel. Dropping the receiver means sends will fail, but we don't care.
```